server/db/models/orderProduct.js

```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const db = require('../db');
+
+const OrderProduct = db.define('orderProduct',{
+	quantity: {
+		type: Sequelize.INTEGER,
+		where:{
```

Comment:
I would not include a where query in your model definition. If you want to make sure something is not less than 0, I would look into validators with min and max.